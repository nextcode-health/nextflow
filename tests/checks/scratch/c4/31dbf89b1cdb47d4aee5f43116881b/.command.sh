#!/bin/bash -ue
echo alpha memry: 7 GB
echo alpha queue: big
