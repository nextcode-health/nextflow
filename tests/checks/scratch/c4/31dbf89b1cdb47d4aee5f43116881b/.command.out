alpha memry: 7 GB
alpha queue: big
